src/page.ts

```diff
@@ -85,6 +85,12 @@ export interface DappeteerPage<P = unknown> {
     name: string,
     callback: Function | { default: Function }
   ): Promise<void>;
+
+  type(
+    selector: string,
+    text: string,
+    options?: { delay: number }
+  ): Promise<void>;
 }
```

Comment:
There is already `type` inside `DappeteerElementHandle`
https://github.com/ChainSafe/dappeteer/blob/d7c51f948d01230c04603e289375af03e872289d/src/element.ts#L1-L11

Reply:
I see, but honestly, I like to have a "type" method on top level as well, I think the code looks more intuitive:
```js
      const accountNameInput = await page.$(".new-account-create-form__input");
      await accountNameInput.type(accountName);
```
or 
```js
await page.type(".new-account-create-form__input", accountName);
```

Reply:
open it as a question for other members to see and left a opinion
cc @irubido @mpetrunic

Reply:
👍